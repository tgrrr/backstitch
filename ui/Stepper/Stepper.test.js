import React from 'react'
import renderer from 'react-test-renderer'
import { Stepper } from '.'

it('Stepper: default', () => {
    const component = renderer.create(
        <Stepper>
            <>
                Stepper
            </>
        </Stepper>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
