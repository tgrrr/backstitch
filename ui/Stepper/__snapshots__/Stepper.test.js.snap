// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stepper: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Stepper "
    data-testid="Stepper"
  >
    <div
      className="MuiPaper-root MuiStepper-root MuiStepper-horizontal MuiPaper-elevation0"
    >
      Stepper
    </div>
  </div>
</div>
`;
